Client/TabManagement/TabMigrationUtility.swift

```diff
@@ -93,7 +93,9 @@ class DefaultTabMigrationUtility: TabMigrationUtility {
 
             let tabData = TabData(id: savedTabUUID,
                                   title: savedTab.title,
-                                  siteUrl: savedTab.url?.absoluteString ?? "",
+                                  // Ecosia: `savedTab.url` is sometimes not there after migration,
+                                  // so we fallback to the last url from the session data history
+                                  siteUrl: savedTab.url?.absoluteString ?? savedTab.sessionData?.urls.last?.absoluteString ?? "",
```

Comment:
Small suggestion to keep things consistent with the commenting.
```suggestion
                                  /* Ecosia: `savedTab.url` is sometimes not there after migration,
                                     so we fallback to the last url from the session data history
                                     siteUrl: savedTab.url?.absoluteString ?? "",
                                     */
                                  siteUrl: savedTab.url?.absoluteString ?? savedTab.sessionData?.urls.last?.absoluteString ?? "",
```